src/containers/HomeRevamp.js

```diff
@@ -241,7 +241,7 @@ function Home(props: HomeProps): Node {
     if (lovelaceDelegation > limit) return pools;
 
     return pools.filter((item) => {
-      return item != null && Number(item.total_stake) + lovelaceDelegation < limit;
+      return true; item != null && Number(item.total_stake) + lovelaceDelegation < limit;
```

Comment:
Is this wanted?

Reply:
oh heck! the entire function were supposed to be removed actually 🙌